test/alice.txt

```diff
@@ -1,2 +1,14 @@
 Alice was beginning to get very tired of sitting by her sister on the bank, and of having nothing to do: once or twice she had peeped into the book her sister was reading, but it had no pictures or conversations in it, “and what is the use of a book,” thought Alice “without pictures or conversations?”
+Alice was beginning to get very tired of sitting by her sister on the bank, and of having nothing to do: once or twice she had peeped into the book her sister was reading, but it had no pictures or conversations in it, “and what is the use of a book,” thought Alice “without pictures or conversations?”
```

Comment:
We have files of many lengths - please don't change Alice! You can use 'coding.txt' if you need something longer.

Reply:
I just wanted something that's just under 4k, but will change this back.